### ISSUE TITLE
Not seeing expected change to client.getMultirotorState().landed_state after takeoff and land

### ISSUE THREAD

**Issue by jasonhbartlett** [Microsoft/AirSim#1776]
Here is my simple program to test control of the drone:
```python
import setup_path
import airsim
import numpy as np
import os
import tempfile
import pprint
import readchar
import time

# connect to the AirSim simulator
client = airsim.MultirotorClient()
client.confirmConnection()
client.enableApiControl(True)
client.armDisarm(True)

state = client.getMultirotorState()
s = pprint.pformat(state)
print("state: %s" % s)

print("Press any key to begin. Once started, press Q to quit.")
cmd = str(readchar.readkey()).lower()
#x = input()
cmd = "b"
while not cmd == "q":
    landed = client.getMultirotorState().landed_state
    print("Landed = ", client.getMultirotorState().landed_state)

    if landed == airsim.LandedState.Landed:
        # Provide user options
        print("Currently landed...")
        print("--Press X to takeoff")
        print("--Press R to reset")

        # Read user input
        cmd = str(readchar.readkey()).lower()

        # Print user input
        print(str(cmd))

        # Respond to user input
        if cmd == "x":
            print("taking off...")
            client.armDisarm(True)
            client.takeoffAsync().join()
            client.hoverAsync().join()
        elif cmd == "r": 
            client.reset()
            time.sleep(3)
            client.enableApiControl(True)
        elif cmd == "q": 
            print("Exiting program.")
        else:
            print("Not a valid selection.")

    else:
        # Provide user options
        print("Currently in flight...")
        print("--Press H to hover")
        print("--Press L to land")
        print("--Press R to reset")

        # Read user input
        cmd = str(readchar.readkey()).lower()

        # Print user input
        print(str(cmd))

        # Respond to user input
        if cmd == "l":
            print("landing...")
            client.landAsync().join()
            print("The drone has landed.")
        elif cmd == "h": 
            print("hovering...")
            client.hoverAsync().join()
        elif cmd == "r": 
            client.reset()
            time.sleep(3)
            client.enableApiControl(True)
        elif cmd == "q": 
            print("Exiting program.")
        else:
            print("Not a valid selection.")
```
For Unreal I expected the call to client.takeoffAsync().join() and client.hoverAsync().join() would result in subsequent call of landed = client.getMultirotorState().landed_state to equal 1.
Instead I get:
![image](https://user-images.githubusercontent.com/22244576/52380760-14b09800-2a3d-11e9-9e54-b814bca0c435.png)

For Unity I DO get the expected behavior after client.takeoffAsync().join() and client.hoverAsync().join() BUT not the expected behavior after the call to client.landAsync().join().
Instead I get:
![image](https://user-images.githubusercontent.com/22244576/52380776-209c5a00-2a3d-11e9-9c0b-4fd6d01082ed.png)

Bottom line:
Shouldn't client.takeoffAsync().join() result in client.getMultirotorState().landed_state = 1
and 
Shouldn't client.landAsync().join() result in client.getMultirotorState().landed_state = 0 
?

**Comment by blakermchale** [comment]
Is there any update on this bug? I was recently using `client.landAsync().join()` and could never get the drone to switch to `client.getMultirotorState().landed_state = 0` after the landing completed
